### ISSUE TITLE
Jenkins settings are taken only from the first folder in the workspace

### ISSUE THREAD

**Issue by pm-energid** [alefragnani/vscode-jenkins-status#93]
Type: <b>Bug</b>

I'm using VS code on a Windows 11 host and opening a remote workspace (in Debian 11 on a docker container in WSL2). It's a multi-project workspace, where each project folder is actually a symlink. I created a .jenkins file in the first project folder of my workspace and it seems to work fine. However, I keep getting the error message "Error while retrieving Jenkins settings" periodically. After further testing, I found out that the extension isn't able to find the .jenkins file from other folders in my workspace and only in the first folder of the workspace.

Extension version: 4.4.1
VS Code version: Code 1.73.1 (6261075646f055b99068d3688932416f2346dd3b, 2022-11-09T04:27:29.066Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No
Remote OS version: Linux x64 5.15.79.1-microsoft-standard-WSL2

<details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
pyindex848cf:30577861
nodejswelcome1:30587005
2e4cg342:30602488
gswce1:30612156
iaj6b796:30613358
dbltrim-noruby:30604474
89544117:30613380
fim-prod:30623723

```

</details>

**Comment by pm-energid** [comment]
![image](https://user-images.githubusercontent.com/14093541/206723428-5a669f10-1381-4a54-ab7a-373b9ee7c32b.png)

**Comment by alefragnani** [comment]
Hi @pm-energid ,

I suppose that message is displayed because you are using `symlink`, as multi-root workspaces should works just fine.

I'm not a fan of `symlinks` (but I know some people love it), and I never tested the extension against it. I had to change another extension to support it couple of months ago and I remember some file operations needs _special attention_. Maybe that's the case.

Hope this helps

**Comment by pm-energid** [comment]
I agree... That may very well be the case. Using symlinks for my project repositories saves me a lot of disk space. I work with multiple inter-dependent repositories and on different branches for different projects. Many of the projects do share the same branches too. So, I setup my projects via git workdirs and symlinks to these workdirs.
